### PR TITLE
readme - update instantiation instructions

### PR DESCRIPTION
**Pull request by kumavis**
Closes https://github.com/Level/memdown/issues/98

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,9 @@
 ```js
 // Note that if multiple instances point to the same location,
 // the db will be shared, but only per process.
-var levelup = require('levelup')
-var db = levelup('/some/location', { db: require('memdown') })
+const levelup = require('levelup')
+const memdown = require('memdown')
+const db = new levelup(new memdown())
```
Comment (vweevers) [inline README.md]: Can you put a newline after the require statements, and remove `new` (2x)?

Comment (juliangruber) [reply README.md]: 👍

**Comment by ralphtheninja** [comment]
Should really have made this PR to the `v2` branch but no biggie.

**Comment by kumavis** [comment]
@vweevers @ralphtheninja updated and retargeted to v2 (new github feature)

**Comment by juliangruber** [comment]
Thank you for your contribution, @kumavis!

**Pull request merged**